vlsi/tutorial.mk

```diff
@@ -0,0 +1,36 @@
+#########################################################################################
+# makefile variables for Hammer tutorials
+#########################################################################################
+tutorial ?= none
+
+# TODO: eventually have asap7 commercial/openroad tutorial flavors
+ifeq ($(tutorial),asap7)
+	tech_name         ?= asap7
+	CONFIG			  ?= TinyRocketConfig
+	TOOLS_CONF	      ?= example-tools.yml
+	TECH_CONF		  ?= example-asap7.yml
+	INPUT_CONFS		  ?= $(EXTRA_CONFS) $(TOOLS_CONF) $(TECH_CONF)
+	VLSI_OBJ_DIR	  ?= build-asap7-commercial
+endif
+
+ifeq ($(tutorial),sky130-commercial)
+	tech_name         ?= sky130
+	CONFIG			  ?= TinyRocketConfig
+	TOOLS_CONF	      ?= example-tools.yml
+	TECH_CONF		  ?= example-sky130.yml
+	DESIGN_CONF		  ?= example-designs/sky130-commercial.yml
+	EXTRA_CONFS       ?= $(if $(filter $(TOP),Rocket RocketTile), example-designs/sky130-rocket.yml, )
```

Comment:
Isn't this setting top to the Rocket module, not ChipTop of TinyRocketConfig?

Reply:
I changed this slightly, but basically this is supposed to check if VLSI_TOP is Rocket, and if so specify sky130-rocket.yml to set a higher clock freq and smaller footprint (and not place any macros). The thinking was Rocket is much easier to use for pipe-cleaning than the full ChipTop.